packages/reporter/src/std/StdReporter.ts

```diff
@@ -9,6 +9,7 @@ export class StdReporter implements Reporter {
     Severity,
     (...x: unknown[]) => string
   > = {
+    [Severity.CRITICAL]: chalk.redBright,
```

Comment:
Please add a new test case for logging critical severity issues to stderr. For details please refer to https://github.com/NeuraLegion/sectester-js/blob/bc05852b5252e71352a11e3f4e684f95641ab0f1/packages/reporter/src/std/StdReporter.spec.ts#L63